hugegraph-test/src/main/java/org/apache/hugegraph/api/MetricsApiTest.java

```diff
@@ -165,8 +190,8 @@ public void testMetricsBackend() {
                     String key = (String) e.getKey();
                     value = e.getValue();
                     Assert.assertTrue(String.format(
-                                      "Expect map value for key %s but got %s",
-                                      key, value),
+                                              "Expect map value for key %s but got %s",
```

Comment:
prefer to keep the origin alignment

Reply:
not addressed: prefer to keep the origin alignment